src/macros.rs

```diff
@@ -150,7 +150,7 @@ macro_rules! libc_bitflags {
     // (non-pub) Entry rule.
     (
         $(#[$attr:meta])*
-        flags $BitFlags:ident: $T:ty {
+        pub flags $BitFlags:ident: $T:ty {
```

Comment:
The macro `libc_bitflags` was broken because `bitflags 0.4` did not support non-public flags.